src/index.js

```diff
@@ -1,2 +1,104 @@
-/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+import scoreboard from './modules/scoreboard.js';
 import './index.css';
+
+const submitButton = document.querySelector('.submit');
+const nameInput = document.querySelector('.name');
+const scoreInput = document.querySelector('.score');
+
+// Define a function to store game ID in local storage
+const setGameId = (gameId) => {
+  localStorage.setItem('gameId', gameId);
+};
+
+// Define a function to get game ID from local storage
+const getGameId = () => localStorage.getItem('gameId');
+
+const createGame = async () => {
+  try {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        name: 'My cool new game',
+      }),
+    });
+    const data = await response.json();
+    const gameId = data.result.split(': ')[1];
+
+    setGameId(gameId); // Store the game ID in local storage
+  } catch (error) {
+    console.error('Error creating game:', error);
+  }
+};
+createGame();
```

Comment:
- [ ] kindly make you have only one game this will prevent your data to lose once the page is refreshed current when I reload the page I can't get the previous scores that I created. Kindly fix this issue.